File: database/port/ActionMatch.java
package database.port;

//[using]

@Table(name = "action_match")
class ActionMatch
{

    @Column(name = "id_action_match")
    public int idActionMatch;    

    @Column(name = "valeur")
    public int valeur;    

    @Column(name = "date_action")
    public java.sql.Timestamp dateAction;    

    @Column(name = "id_match")
    public String idMatch;    

    @Column(name = "id_equipe_joueur")
    public String idEquipeJoueur;    

    @Column(name = "id_action")
    public String idAction;    



    public void setIdActionMatch(int value){
        this.idActionMatch = value;
    }
    public int getIdActionMatch(){
        return this.idActionMatch;
    }

    public void setValeur(int value){
        this.valeur = value;
    }
    public int getValeur(){
        return this.valeur;
    }

    public void setDateAction(java.sql.Timestamp value){
        this.dateAction = value;
    }
    public java.sql.Timestamp getDateAction(){
        return this.dateAction;
    }

    public void setIdMatch(String value){
        this.idMatch = value;
    }
    public String getIdMatch(){
        return this.idMatch;
    }

    public void setIdEquipeJoueur(String value){
        this.idEquipeJoueur = value;
    }
    public String getIdEquipeJoueur(){
        return this.idEquipeJoueur;
    }

    public void setIdAction(String value){
        this.idAction = value;
    }
    public String getIdAction(){
        return this.idAction;
    }

   
}

File: database/port/ActionMatch.cs
namespace database.port;

//[using]

[Table(name = "action_match")]
class ActionMatch
{

    [Column(name = "id_action_match")]
    public Int idActionMatch { get; set; }    

    [Column(name = "valeur")]
    public Int valeur { get; set; }    

    [Column(name = "date_action")]
    public DateTime dateAction { get; set; }    

    [Column(name = "id_match")]
    public String idMatch { get; set; }    

    [Column(name = "id_equipe_joueur")]
    public String idEquipeJoueur { get; set; }    

    [Column(name = "id_action")]
    public String idAction { get; set; }    



    public void SetIdActionMatch(Int value){
        this.idActionMatch = value;
    }
    public Int GetIdActionMatch(){
        return this.idActionMatch;
    }

    public void SetValeur(Int value){
        this.valeur = value;
    }
    public Int GetValeur(){
        return this.valeur;
    }

    public void SetDateAction(DateTime value){
        this.dateAction = value;
    }
    public DateTime GetDateAction(){
        return this.dateAction;
    }

    public void SetIdMatch(String value){
        this.idMatch = value;
    }
    public String GetIdMatch(){
        return this.idMatch;
    }

    public void SetIdEquipeJoueur(String value){
        this.idEquipeJoueur = value;
    }
    public String GetIdEquipeJoueur(){
        return this.idEquipeJoueur;
    }

    public void SetIdAction(String value){
        this.idAction = value;
    }
    public String GetIdAction(){
        return this.idAction;
    }

   
}
